src/run_build.py

```diff
@@ -41,6 +42,12 @@ def main() -> int:
 
     output_dir = BuildOutputDir(manifest.build.filename, args.distribution).dir
 
+    if args.incremental:
```

Comment:
please add test.

Reply:
This is a placeholder for the next PR and it's just logging a list of plugins for rebuilding. I have complete tests for all those functions within `buildIncremental` class.

Reply:
It still is part of the code flow and should be tested, if not in this PR then in subsequent PRs.

Reply:
Will absolutely add tests in my following PR.